README.md

```diff
@@ -20,14 +20,14 @@ To run SOIT manually :
    - [ ] ` export SPACEPSWD=<password>`
 3. Update inputs and run the following: 
 ```bash
-docker run --env SPACEUSER --env SPACEPSWD --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/ \
-icefloetracker-fetchdata:main \
+docker run --env SPACEUSER --env SPACEPSWD --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/icefloetracker-fetchdata:main \
```

Comment:
The extra space in line 23 was giving me an uninformative error from docker "docker: invalid reference format". Deleting the space between `brownccv/` and `icefloetracker-fetchdata:main` made it so it would run.

Reply:
Good find!